api/v1/types.go

```diff
@@ -79,11 +79,13 @@ type AccountList struct {
 
 // Account is the API response for GetAccount.
 type Account struct {
-	Address   string `json:"address"`
-	Nonce     uint64 `json:"nonce"`
-	Available uint64 `json:"available"`
-	Escrow    uint64 `json:"escrow"`
-	Debonding uint64 `json:"debonding"`
+	Address                     string `json:"address"`
+	Nonce                       uint64 `json:"nonce"`
+	Available                   uint64 `json:"available"`
+	Escrow                      uint64 `json:"escrow"`
+	Debonding                   uint64 `json:"debonding"`
+	DelegationsBalance          uint64 `json:"delegations_balance,omitempty"`
+	DebondingDelegationsBalance uint64 `json:"debonding_delegations_balance,omitempty"`
```

Comment:
I don't fully understand why these are different from `Escrow` and `Debonding` fields rather than just replacing them?

Reply:
Hmm, I think we can remove them if we can expect `validator` information exclusively from the `/validators` endpoint?

Currently, `Escrow` returns active escrow pool balance, and `Debonding` returns debonding pool balance. `DelegationsBalance` and `DebondingDelegationsBalance` return the delegated estimates.